cloud/cluster/state.go

```diff
@@ -52,6 +56,7 @@ func (s *state) setAndDiff(newState []Node) []NodeUpdate {
 	// reset nodes map, assign to new state
 	s.nodes = make(map[NodeId]Node)
 	for _, n := range newState {
+		log.Infof("Node %s included in new cluster state", n)
```

Comment:
instead of printing this, how about printing:
number of nodes added, number of nodes removed, then also print same info (number of nodes added, number of nodes removed) from cluster_state.go?

wdyt?

Reply:
added